app/server/mocks/prisma/client.ts

```diff
@@ -0,0 +1,5 @@
+import { PrismaClient } from '../../src/__generated__/prisma';
+
+// Error occurs unless prisma is a default export
```

Comment:
Maybe "Error occurs due to non-prytaneum code unless prisma is the default export"